Lib/test/test_sqlite3/test_userfunctions.py

```diff
@@ -405,19 +405,19 @@ def test_func_deterministic_keyword_only(self):
     def test_function_destructor_via_gc(self):
         # See bpo-44304: The destructor of the user function can
         # crash if is called without the GIL from the gc functions
-        dest = sqlite.connect(':memory:')
-        def md5sum(t):
-            return
+        with memory_database() as dest:
+            def md5sum(t):
```

Comment:
Function should be outside of the context manager, I guess.

Reply:
Good catch!